lib/PuppeteerSharp/Frame.cs

```diff
@@ -513,7 +512,7 @@ public Task<string> GetContentAsync()
         /// <returns>Task.</returns>
         /// <seealso cref="Page.SetContentAsync(string)"/>
         public Task SetContentAsync(string html)
-            => EvaluateFunctionAsync(@"html => {
+            => EvaluateFunctionAsync<object>(@"html => {
```

Comment:
Same question here. `EvaluateFunctionAsync(@"html => {` should still be valid

Reply:
This is needed as it defaults to JToken which can't be converted implicitly to string

Reply:
But `SetContentAsync` is a void method.

Reply:
Ah yes, the line number was pointing me at GetContentAsync